src/components/EditDialog.tsx

```diff
@@ -123,16 +123,14 @@ export default function EditDialog({ data }: Props) {
   const handleSubmit = async () => {
     let modifiedDate = dateValue;
     if (enableDate) {
-      modifiedDate = changeDateFormat(
-        dateValue.$d.toString().slice(4, 15).replace(/ /g, "-")
-      );
+      modifiedDate = dateValue.toISOString();
+      console.log(modifiedDate);
```

Comment:
Bad idea to leave "console.log()" calls in your production code.

Please be careful from next time onward.